external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/ConsumerStrategy.scala

```diff
@@ -83,10 +86,11 @@ case class SubscribeStrategy(topics: Seq[String]) extends ConsumerStrategy {
 /**
  * Use a regex to specify topics of interest.
  */
-case class SubscribePatternStrategy(topicPattern: String) extends ConsumerStrategy {
+case class SubscribePatternStrategy(topicPattern: String,
```

Comment:
These classes are technically public, but are they used by non-Spark code? Because if they are this would be an API breakage. If they're not meant to be used by non-Spark code, they probably should be `private[something]`.

Reply:
Presume its coming from the old [DStreams API](https://github.com/gaborgsomogyi/spark-dstream-secure-kafka-app/blob/161bf02eb3677aac604d63499041f72231d0e371/src/main/scala/com/cloudera/spark/examples/DirectKafkaWordCount.scala#L78) since it's different in [Structured Streaming](https://github.com/gaborgsomogyi/spark-structured-secure-kafka-app/blob/e2c85aba7f2cfb8af3656b5aff25cf13404933e9/src/main/scala/com/cloudera/spark/examples/StructuredKafkaWordCount.scala#L60). Will hide them...

Reply:
Done.